extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/graal/SQLServerDatabaseTypeSubstitutions.java

```diff
@@ -15,9 +16,11 @@
 public final class SQLServerDatabaseTypeSubstitutions {
```

Comment:
I wonder if the substitution class can be removed. @Vinche59 Can you give it a try?

Reply:
Yeah I think we would just get rid of this class entirely.

Reply:
I just tested locally. Removing the class entirely resolves the problem

Reply:
Nice!

Reply:
Sorry was afk this afternoon.
I confirm that removing the class entirely fixes also my problem.
Thank you.